doc/user_guide/getting_started.rst

```diff
@@ -49,13 +49,14 @@ can be started as follows:
 
 
 
-.. versionadded:: 0.8.1
+.. versionchanged:: 0.8.3
```

Comment:
Never looked into that too much myself. I suppose the way we actually use it is just having the text there. Also, if we ever end up using it, [this](http://sphinx-doc.org/builders.html?highlight=changes#sphinx.builders.changes.ChangesBuilder)